### PR TITLE
Bug #48

### PR DESCRIPTION
**Pull request by YoungFaithful**
Addressing bug #48

### CODE REVIEW

File: src/clustering/run_clust.jl [hunk 1]
```diff
@@ -38,7 +38,7 @@ function run_clust(data::ClustData;
 
     # set configuration file
     clust_config = set_clust_config(;norm_op=norm_op, norm_scope=norm_scope, method=method, representation=representation, n_clust=n_clust, n_seg=n_seg, n_init=n_init, iterations=iterations, attribute_weights=attribute_weights)
-    
+
```
Comment (YoungFaithful) [inline src/clustering/run_clust.jl]: Don't know what the editor does there...

**Comment by YoungFaithful** [comment]
CEP testing was successful with those changes.

**Comment by YoungFaithful** [comment]
I think we should really think about more detailed testing of the clustering-results like in CapacityExpansion

**Comment by holgerteichgraeber** [comment]
> 
> 
> I think we should really think about more detailed testing of the clustering-results like in CapacityExpansion

Agreed! I am working on enhancing the documentation right now, and testing is the next step before we can submit to JOSS.

**Comment by holgerteichgraeber** [comment]
@YoungFaithful do you know why the test failed?

**Comment by holgerteichgraeber** [comment]
> 
> 
> @YoungFaithful do you know why the test failed?

Ok, I guess just because it was merged before tests had finished..

**Pull request merged**